plugins/lime-plugin-fbw/src/containers/Scan.js

```diff
@@ -1,25 +1,32 @@
 import { h } from 'preact';
 import { useState, useEffect } from 'preact/hooks';
 
-import '../style.less';
+import style from '../style.less';
 
 import { Trans, t } from '@lingui/macro';
 import { Loading } from 'components/loading';
 import Toast from 'components/toast';
 import { isValidHostname, slugify } from 'utils/isValidHostname';
 import { useBoardData } from 'utils/queries';
 import { useSearchNetworks, useSetNetwork, useGetNetworks } from '../FbwQueries';
+import { List, ListItem } from 'components/list';
+import { SignalBar } from '../../../lime-plugin-align/src/components/signalBar'; // todo(kon): implement common component
+
 
 export const Scan = ({ toggleForm, setExpectedHost, setExpectedNetwork }) => {
```

Comment:
This component got huge, It found it difficult to follow the code.
I think that a good first step toward making it more clear could be to separate both screens: the one listing the scanned networks, and the one in which you configure a hostname, in two different files.